### PR TITLE
Make `eip4844` green again

### PR DESCRIPTION
**Pull request by terencechain**

### CODE REVIEW

File: beacon-chain/p2p/testing/p2p.go [hunk 1]
```diff
@@ -48,11 +48,6 @@ type TestP2P struct {
 	LocalMetadata   metadata.Metadata
 }
 
-func (p *TestP2P) BroadcastBlob(ctx context.Context, subnet uint64, blobSidecar *ethpb.SignedBlobSidecar) error {
-	//TODO implement me
-	panic("implement me")
-}
-
```
Comment (kasey) [inline beacon-chain/p2p/testing/p2p.go:55]: Why did you delete these? These methods were missing and causing the test types to not fulfill the interface.

Comment (terencechain) [reply beacon-chain/p2p/testing/p2p.go:55]: I deleted the duplications. I think they got added twice by two different PRs

Comment (kasey) [reply beacon-chain/p2p/testing/p2p.go:55]: gotcha ty
